full spelling and grammar check/and correction (#70)
MillenniumMachines/millenniummachines.github.io: docs/millennium-os/manual/chapters/10_introduction.md

If you don't have the hardware for one or more of these features or you disable the feature, MillenniumOS will fall back on other, manual methods to achieve the same goal.

This allows us to use the same gcode dialect for machines that have any combination of feature support.

> This may be hard to visualize, so here are some examples:

Reviewer: Americanisation :)

### Toolsetter and Touch Probe Disabled

You will be guided through manual work-piece probing when the machine needs to know the origin of a work-piece. Every time a tool is changed, you will be guided through a manual process to re-zero the origin in the Z-Axis, as the tool length will have changed.

> You can do everything, but it will take a while as this will involve manual probing at every step.

Reviewer: ```suggestion
You can do everything required for job set up via this process, but it will take a long time as each step will involve manual probing methods.
```

### Only Toolsetter Enabled

You will still be guided through manual work-piece probing, however, when you change tools during the machining process, we will use the toolsetter to calculate the relative length difference between the tools and compensate for this automatically.

> This will save time if your code contains multiple operations with tool changes.

Reviewer: ```suggestion
This will save time if your code contains multiple operations with tool changes, and avoids the need to update the Z-Axis origin after changing tools.
```

### Only Touch Probe Enabled

> You will be guided through an automated work-piece probing system, similar to the manual probing system but will use your touch probe for the actual probing moves. This will speed up the probing process *considerably*.

Reviewer: ```suggestion
You will be guided through an automated work-piece probing system, similar to the manual probing system but using your touch probe for the actual probing moves. This will speed up the probing process *considerably*.
```

> You will still have to re-probe the Z-height of the workpiece manually after every tool change as we have no way to calculate the relative length of the tools.

Reviewer: ```suggestion
You will still have to re-probe the Z-Axis origin of the workpiece manually after every tool change as we have no way to calculate the relative length of the tools.
```

### Both Touch Probe and Toolsetter Enabled

You will have the best of both worlds. There will no longer be any need for manual probing routines, and aside from the manual steps you take to position the touch probe at the start of a probing procedure, and changing the tools themselves, everything else will be automated.

> This will save you the most time and is the most accurate way of using MillenniumOS.

Reviewer: ```suggestion
This will save you the most time and is the most accurate way to use MillenniumOS.
```
